### PR TITLE
Fixed a malformed table

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR fixes a malformed table in the firewall prerequisites for ROSA. I also removed some leftover merge conflict information.

PREVIEW: 

* https://deploy-preview-42993--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-aws-prereqs#osd-aws-privatelink-firewall-prerequisites

### CODE REVIEW

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 1]
```diff
@@ -57,8 +57,8 @@ This section provides the necessary details that enable you to control egress tr
 |Provides access to the odo CLI tool that helps developers build on OpenShift and Kubernetes.
 
 |`console.redhat.com`
-|Required. Allows interactions between the cluster and OpenShift Console Manager to enable functionality, such as scheduling upgrades.
 |443, 80
+|Required. Allows interactions between the cluster and OpenShift Console Manager to enable functionality, such as scheduling upgrades.
 
 |`sso.redhat.com`
```
Comment (michaelryanpeter) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:63]: Does it matter that line 65 still has `cloud.redhat.com`?

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 2]
```diff
@@ -33,8 +33,8 @@ This section provides the necessary details that enable you to control egress tr
 |Provides core container images.
 
 |`sso.redhat.com`
-|Required. The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com` to  download the pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, chargeback reporting, and so on.
 |443, 80
+|Required. The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com` to  download the pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, chargeback reporting, and so on.
```
Comment (michaelryanpeter) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:37]:
```suggestion
|Required. The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com` to download the pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, chargeback reporting, and so on.
```

**Review by michaelryanpeter — COMMENTED** [review]
A nit and a question. In all these PRs, the `{op-system-first}` variable is not rendering.

**Review by stevsmit — APPROVED** [review]

**Pull request merged**